views/js/component/diagnostic/diagnosticLoader.js

```diff
@@ -69,7 +70,20 @@ define([
                             factoryConfig = componentConfig[factoryName];
                             factoryConfig.controller = componentConfig.controller;
 
-                            factory(self.getElement(), factoryConfig);
+                            var componentInstance = factory(self.getElement(), factoryConfig);
```

Comment:
I don't see the benefit of this variable as it is only used once. Moreover, when using the `var` keyword, please always put it on top, to reflect hoisting.

Reply:
ok, since this is a backport, nothing to change. But that means a review has been messed up.